Sources_v3/StreamChatUI/ChatChannel/MessageComposerInputAccessoryViewController.swift

```diff
@@ -128,7 +128,9 @@ open class MessageComposerInputAccessoryViewController<ExtraData: ExtraDataTypes
     }
     
     func setupInputView() {
-        inputView = composerView
+        inputView?.embed(composerView)
+        inputView?.translatesAutoresizingMaskIntoConstraints = false
+        inputView?.allowsSelfSizing = true
```

Comment:
Under which circumstances can `inputView` be `nil`? We should check for this and log an error. Otherwise, this will silently fail and we will have no idea.